.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Prettierrc seems to be more standard than the JSON config. The ignore should also help, especially with build times